recipes/conversation_view/feed_pill/feed_item_pill.test.js

```diff
@@ -0,0 +1,121 @@
+  };
+
+  const MOCK_ARIA_LABEL = 'Click to expand';
+  const MOCK_ICON_NAME = 'Video';
+
+  const baseProps = {
+    iconName: MOCK_ICON_NAME,
+    title: 'This meeting has ended',
+    ariaLabel: MOCK_ARIA_LABEL,
+    buttonClass: '',
+    toggleable: true,
+  };
+  const baseAttrs = {};
+  const baseSlots = {
+    content: '<div data-qa="content-element"> content </div>',
+  };
+  const baseProvide = {};
+
+  let mockProps = {};
+  let mockAttrs = {};
+  let mockSlots = {};
+  let mockProvide = {};
+
+  const updateWrapper = () => {
+    wrapper = mount(DtRecipeFeedItemPill, {
+      propsData: { ...baseProps, ...mockProps },
+      attrs: { ...baseAttrs, ...mockAttrs },
+      slots: { ...baseSlots, ...mockSlots },
+      global: {
+        provide: { ...baseProvide, ...mockProvide },
+      },
+    });
+    _setChildWrappers();
+  };
+
+  // Setup
+  beforeEach(function () {
+    updateWrapper();
+  });
+
+  // Teardown
+  afterEach(function () {
+    mockProps = {};
+    mockAttrs = {};
+    mockSlots = {};
+    mockProvide = {};
+  });
+
+  describe('Presentation Tests', function () {
+    describe('Default render', function () {
+      it('should exist', () => {
+        expect(wrapper.exists()).toBe(true);
+      });
+
+      it('should render a feed item pill', () => {
+        expect(feedItemPill.exists()).toBeTruthy();
+        expect(icon.exists()).toBe(true);
+        expect(icon.attributes('data-name')).toBe(MOCK_ICON_NAME);
+        expect(wrapper.find('[data-qa="content-element"]').exists()).toBe(false);
+      });
+    });
+  });
+
+  describe('Accessibility Tests', function () {
+    describe('Default Render', function () {
+      it('should render a reaction button', () => {
+        expect(feedItemPill.attributes('aria-label')).toBe(MOCK_ARIA_LABEL);
+      });
+    });
+  });
+
+  describe('Interactivity Tests', function () {
+    describe('Toggle feed item pill', function () {
+      it('Should emit a click event', async () => {
+        await feedItemPill.trigger('click');
+        await wrapper.vm.$nextTick();
+        expect(wrapper.find('[data-qa="content-element"]').exists()).toBe(true);
+      });
+
+      describe('toggleable false', function () {
+        beforeAll(() => {
+          mockProps = {
+            toggleable: false,
+          };
+        });
+
+        it('should exist', () => {
+          expect(wrapper.exists()).toBe(true);
+        });
+
+        it('should not respond to clicks', () => {
+          expect(feedItemPill.exists()).toBeTruthy();
+          expect(icon.exists()).toBe(true);
+          expect(icon.attributes('data-name')).toBe(MOCK_ICON_NAME);
+          expect(wrapper.find('[data-qa="content-element"]').exists()).toBe(false);
+        });
+      });
+    });
+
+    // Test not working
+    describe.skip('Hover Feed Item Pill event', function () {
+      it('should show a different icon', async () => {
+        await feedItemPill.trigger('focus');
```

Comment:
This is not working because the event is `focusin` rather than `focus`

Reply:
🤦 yeah that works now